src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,20 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.service.DriverService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+        Driver driver = new Driver(null, "Dominic", "MA-753159");
+        driverService.create(driver);
+        System.out.println(driverService.get(1L));
+        driver.setLicenseNumber("MA-987654321");
+        driverService.update(driver);
+        driverService.getAll().forEach(System.out::println);
+        driverService.delete(1L);
     }
```

Comment:
it would be better to also test `ManufacturerService`

Reply:
Thanks. ManufacturedService was provided with assignment and I tested it in the previos task)